public/js/checkin.js

```diff
@@ -14,8 +14,29 @@ window.HLRDESK.init.checkin = function() {
     return $(node).data("date")||$(node).text()
   }
   $(".checkIN").tablesorter( {textExtraction: myTextExtraction} );
-  
+  function checkInItems(evt){
+    evt.preventDefault();
+    var selected = document.querySelectorAll('.selected');
+    for(var i = 0; i<selected.length; i++){
+      selected[i].parentNode.removeChild(selected[i]);
```

Comment:
@Luke1298 unfortunately the JavaScript DOM specification doesn't have a way to remove an element in a simple way. You have to remove it from a parent element.